nearlib/test/integration.test.js

```diff
@@ -10,8 +11,11 @@ beforeAll(async () => {
     const options = {};
     options.nodeUrl = 'http://localhost:3030';
     options.useAliceAccount = true;
+    keyStore =  new InMemoryKeyStore();
+    options.deps = {
```

Comment:
why wrap into `deps`?
most commonly used options should just be available top-level.

Reply:
non-trivial probability of having a connect method variation that takes deps as params. I want to be able to pass in options.deps rather than copying internals. Also organizing like things together is good.